### PR TITLE
Travis always deploy to beta; ./prod.sh deploy script

### PR DESCRIPTION
**Pull request by toddmedema**
closes https://github.com/ExpeditionRPG/expedition-quest-creator/issues/347

### CODE REVIEW

File: app/Constants.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const packageJson: any = require('../package.json');
 
 export const NODE_ENV = (process && process.env && process.env.NODE_ENV) || 'dev';
-export const API_HOST = (process && process.env && process.env.API_HOST) || '';
+export const API_HOST = (process && process.env && process.env.API_HOST) || 'http://betaquests.expeditiongame.com';
```
Comment (smartin015) [inline app/Constants.tsx]: Shouldn't this be `betaapi.*` now?

File: prod.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+read -p "Did you test on beta? (y/N) " -n 1
+echo
+if [[ $REPLY =~ ^[Yy]$ ]]; then
+  rm -rf dist
+
+  # Build the app - manually configure for production
+  export NODE_ENV='production'
+  export API_HOST='https://api.expeditiongame.com'
+  webpack --config ./webpack.dist.config.js
```
Comment (smartin015) [inline prod.sh]: No `-p` for optimization/production setting?

File: app/Constants.tsx (hunk 1 above)
Comment (toddmedema) [reply app/Constants.tsx]: Ah, yep

File: prod.sh (hunk 2 above)
Comment (toddmedema) [reply prod.sh]: Based on the webpack documentation, we're already doing both things that does (NODE ENV = production and trigger uglifyjs in the dist config); given that, I'd hesitate to add a new flag whose definition and effects may change in the future

Comment (smartin015) [reply prod.sh]: Fair 'nuff.

**Pull request merged**